devsetup/scripts/gen-edpm-compute-node.sh

```diff
@@ -148,6 +148,16 @@ cat <<EOF >../out/edpm/${EDPM_COMPUTE_NAME}.xml
 </domain>
 EOF
 
+cat <<EOF >../out/edpm/tripleocurrent.repo
```

Comment:
oh just have seen that this is currently done in https://github.com/openstack-k8s-operators/install_yamls/blob/master/devsetup/edpm/edpm-play.yaml#L14